gustaf/show.py

```diff
@@ -353,16 +353,14 @@ def make_showable(obj, as_dict=False, **kwargs):
             sb3d_kwargs = {} if isinstance(sb3d_kwargs, bool) else sb3d_kwargs
             vedo_obj.add_scalarbar3d(**sb3d_kwargs)
 
-    elif data_name is not None and vertex_data is None:
-        utils.log.debug(
-            f"No vertex_data named '{data_name}' for {obj}. Skipping"
-        )
+    elif data is not None and vertex_data is None:
+        utils.log.debug(f"No vertex_data named '{data}' for {obj}. Skipping")
 
     # arrow plots - this is independent from data plotting.
-    arrow_data_name = obj.show_options.get("arrow_data", None)
+    arrow_data = obj.show_options.get("arrow_data", None)
```

Comment:
didn't mean to change those, but as it doesn't change any result, I will leave it as it is